src/desktop/components/split_test/running_tests.coffee

```diff
@@ -15,10 +15,24 @@
 # module.exports = {}
 
 module.exports = {
- onboarding_test:
-   key: 'onboarding_test'
-   outcomes:
-     control: 50
-     experiment: 50
-   edge: 'experiment'
+  collect_keyword_search:
```

Comment:
this test has been completed and should be removed from here i believe - a rebase with master should have removed it.